src/components/preview/PersonalInfoItem.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const PersonalInfoItem = ({ title, data, preview }) => {
+  const [hidden, setHidden] = useState(0);
+
+  return (
+    data && (
+      <li>
+        <div>
+          <div>
+            <span>Your {title.charAt(0).toUpperCase() + title.slice(1)}</span>
```

Comment:
If you are uppercasing the first char of label, you can use:

text-transform: capitalize on a single word.
::first-letter { text-transform: uppercase; } for more then one word.

Reply:
this 'preview' folder will be removed in next commit. Its same as the above component